tests/deployment/serverless/test_aws_lambda.py

```diff
@@ -0,0 +1,194 @@
+import os
+from sys import version_info
+
+from mock import MagicMock, patch, Mock, mock_open
+from moto import mock_cloudformation, mock_lambda, mock_s3, mock_apigateway
+from ruamel.yaml import YAML
+
+from bentoml.deployment.serverless.aws_lambda import (
+    generate_aws_lambda_serverless_config,
+    generate_aws_lambda_handler_py,
+    AwsLambdaDeploymentOperator,
+)
+from bentoml.proto.deployment_pb2 import Deployment
+from bentoml.proto.repository_pb2 import Bento, BentoServiceMetadata, GetBentoResponse
+from bentoml.proto.status_pb2 import Status
+from bentoml.deployment.serverless import serverless_utils
+
+
+def test_generate_aws_lambda_serverless_config(tmpdir):
+    python_version = '3.7.2'
+    deployment_name = 'test_deployment_lambda'
+    api_names = ['predict']
+    region = 'us-west-test'
+    namespace = 'namespace'
+    generate_aws_lambda_serverless_config(
+        python_version, deployment_name, api_names, str(tmpdir), region, namespace
+    )
+    config_path = os.path.join(str(tmpdir), 'serverless.yml')
+    yaml = YAML()
+    with open(config_path, 'rb') as f:
+        yaml_data = yaml.load(f.read())
+    assert yaml_data['service'] == deployment_name
+    assert yaml_data['functions']['predict']['handler'] == 'handler.predict'
+    assert yaml_data['provider']['region'] == 'us-west-test'
+
+
+def test_generate_aws_lambda_handler_py(tmpdir):
+    bento_name = 'bento_name'
+    api_names = ['predict', 'second_predict']
+    generate_aws_lambda_handler_py(bento_name, api_names, str(tmpdir))
+
+    import sys
+
+    sys.modules['bento_name'] = Mock()
```

Comment:
remove it after the test?

Reply:
and would be nice to verify `Mock.load` is called, and have it return  an object with `get_service_api('predict')`, that way you can actually call the `predict` api function imported below